### ISSUE TITLE
Bad Android Default Styles

### ISSUE THREAD

**Issue by SpencerLynn** [Purii/react-native-tableview-simple#73]
Using the example on Snack.io, Android styles appear to have some bad margin/padding left/right.

https://snack.expo.io/HJ37Ihvn4

Normal state: Padding on the right seems too much.
![image](https://user-images.githubusercontent.com/5921571/57669178-cae8c800-75be-11e9-84e2-665cf17f6ea8.png)

Pressed state: The left/right margins are darker than the rest.
![image](https://user-images.githubusercontent.com/5921571/57669158-be646f80-75be-11e9-86e2-bb18ce11c318.png)

Just a guess, but based on the darker pressed areas, it seems that margin/padding is off on the content wrapper.

Happy to help any way I can.

Thanks!

**Comment by Purii** [comment]
Appears to be an issue with `SafeAreaView`.
Disabling that implementation seems to fix your issue:

```
<Cell
  cellStyle="Basic"
  title="Pressable w/ accessory"
  accessory="DisclosureIndicator"
  withSafeAreaView={false}
  onPress={() => console.log('Heyho!')}
/>
```

**Comment by Purii** [comment]
Could you please check, if your issues are fixed with v2.1.0?
